docs/UserGuide.adoc

```diff
@@ -226,7 +273,7 @@ Format: `exit`
 
 === Saving the data
 
-Address book data are saved in the hard disk automatically after any command that changes the data. +
+BookInventory data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
 // tag::dataencryption[]
```

Comment:
Good! Might want to fill the part on data encryption in the next update to documentation. 👍